config-templates/aws/aws-efs-csi-driver/1.0.0/README.md

```diff
@@ -1,112 +1,65 @@
-## How to use the template to deploy aws-efs-csi-driver on a Satellite Cluster?
+# AWS EFS CSI Driver
+
+AWS EFS CSI driver implements the CSI specification for container orchestrators to manage the lifecycle of Amazon EFS file systems.
+
+## Prerequisites
 
-### Pre-reqs 
 - Currently only *static provisioning* is supported. This means an [AWS EFS file system](https://docs.aws.amazon.com/efs/latest/ug/gs-step-two-create-efs-resources.html) needs to be created manually on AWS first. After that it can be mounted inside a container as a volume using the driver.
 
+## AWS EBS CSI Driver parameters & how to retrieve them
+
+No parameters required for this template.
```

Comment:
Are there any optional parameters that the user can specify? If so, we should list those here.
```suggestion
No parameters required for this template. 
```

Reply:
There is no optional parameters required for AWS EFS.